docs/guides/setting-up.md

```diff
@@ -110,7 +110,8 @@ When you start Android Studio for the first time it will go through the installa
 
 Once this is complete you should have everything you need for Android development. You can close Android Studio as you won't need it for DoneJS development.
 
-## Mac OS X
+<a id="mac-os-x"></a>
```

Comment:
The title changed, so the ID of the section changed, so this is to keep links pointing to the old section ID working.